server/src/prisma/index.ts

```diff
@@ -158,6 +159,10 @@ type NameChange = Omit<PrismaNameChange, 'reviewContext'> & {
 
 type GroupSocialLinks = Omit<PrismaGroupSocialLinks, 'id' | 'groupId' | 'createdAt' | 'updatedAt'>;
 
+type GroupRoleOrder = Omit<PrismaGroupRoleOrder, 'index'> & {
```

Comment:
You don't need this, I only do this to bigint fields (hack for prisma, don't ask 🤦 )

Just export as normal like `Competition`